code/modules/mob/living/simple_animal/hostile/giant_spider/effects.dm

```diff
@@ -180,4 +181,45 @@
 		A.forceMove(src.loc)
 	..()
 
+
+//Spawns on top of mobs hit with the queen's web projectile
+/obj/effect/overlay/stickyweb
+	name = "sticky web"
+	desc = "A mess of sticky strings."
+	icon = 'icons/effects/effects.dmi'
+	icon_state = "stickyweb"
+	anchored = 1
+	density = 0
+	plane = ABOVE_HUMAN_PLANE
+	layer = CLOSED_CURTAIN_LAYER
+	var/atom/stuck_to = null
+	var/duration = 100
+
+/obj/effect/overlay/stickyweb/Destroy()
+	if(stuck_to)
+		unlock_atom(stuck_to)
+	stuck_to = null
+	..()
+
+/obj/effect/overlay/stickyweb/proc/stick_to(var/atom/A, var/side = null)
+	var/turf/T = get_turf(A)
+	playsound(T, 'sound/weapons/hivehand_empty.ogg', 75, 1)
+
+	if(isliving(A) && !isspace(T))//can't nail people down unless there's a turf to nail them to.
+		stuck_to = A
+		visible_message("<span class='warning'>\the sticky ball splatters over \the [A]'s legs, sticking them to \the [T].</span>")
+		lock_atom(A, /datum/locking_category/buckle)
+
+	spawn(duration)
+		qdel(src)
+
+/obj/effect/overlay/stickyweb/attack_hand(var/mob/user)
+	if (do_after(user,src,15))
```

Comment:
1.5 SECONDS

also it seems weird that you can free yourself by hand but not destroy with a weapon?